webservices/utils.py

```diff
@@ -335,3 +340,15 @@ def create_eregs_link(part, section):
     if section:
         url_part_section += '-' + section
     return '/regulations/{}/CURRENT'.format(url_part_section)
+
+def post_to_slack(message, channel):
```

Comment:
As mentioned before, what if we set an environment variable for the default Slack channel to post to, but keep the parameter as you have to so we can override it.

Reply:
Documenting here: We talked it over and it is easier to have a default that we don't have to configure across all environments, we could do an optional value you pass in the future.